DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PsfSearch/SearchResult.cshtml

```diff
@@ -4,33 +4,38 @@
         @{ Html.EnableClientValidation(false); }
 
         <form action="@Model.BackPageUrl" method="post">
-            <div id="BackLink" class="breadcrumbs">
-                <ol>
-                    <li>
-                        <button type="submit" class="button-link" id="filter-home" formaction="@Model.BackPageUrl">@Model.BackPageUrlText</button>
-                    </li>
-                </ol>
+            <div id="BackLink" class="govuk-breadcrumbs">
+                <button type="submit" class="govuk-back-link" id="filter-home" formaction="@Model.BackPageUrl">@Model.BackPageUrlText</button>
             </div>
             @Html.HiddenFor(m => Model.PreSearchFiltersModel.OptionsSelected)
             @Html.HiddenFor(m => Model.PreSearchFiltersModel.Section.PageNumber)
             @Html.HiddenFor(m => Model.PreSearchFiltersModel.Section.SingleSelectOnly)
         </form>
 
-            <header class="filter-results-header">
-                <h1 class="filter-results-heading">@Model.MainPageTitle</h1>
-                <p class="filter-results-subheading">@Model.SecondaryText</p>
-                <div class="filter-results-count scoped-result-count" data-ga-result-count="@Model.TotalResultCount">@Model.TotalResultsMessage</div>
-            </header>
+        <header class="filter-results-header">
+            <h1 class="govuk-heading-xl">@Model.MainPageTitle<span class="govuk-caption-m">@Model.SecondaryText</span></h1>
 
-            <div id="results">
-                <div class="results-block">
-                    <div class="inner-block">
+            <div class="govuk-warning-text">
+                <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                <strong class="govuk-warning-text__text">
+                    <span class="govuk-warning-text__assistive">Warning</span>
+                    If the job roles you have been matched to are COVID impacted, you should
+                    <a href="#" class="govuk-link">contact our advisers</a> to consider your options and next steps.
```

Comment:
In correct anchor link #

Reply:
Updated the anchor link to contact-us page